docs/DEPRECATIONS.md

```diff
@@ -10,12 +10,14 @@ The format for each entry should be: `<version> <identifier> <description>`.
 
 For example:
 
-- legacy_openssl_provider v0.34.0 OpenSSL legacy provider flag should be removed
+- v0.34.0 legacy_openssl_provider OpenSSL legacy provider flag should be removed
 
 ## To be deprecated
 
+- v0.37.0 strict_env_vars Add deprecation warning for missing environment variable interpolation.
```

Comment:
```suggestion
- v0.35.0 strict_env_vars Add deprecation warning for missing environment variable interpolation.
```

I think you could deprecate it in the next release if you wanted to. This would mean adding it to the upgrade guide.

Reply:
Fixed in 93dff07d5